backend/src/main/java/com/znane_groby/backend/configuration/LoadDatabase.java

```diff
@@ -0,0 +1,135 @@
+package com.znane_groby.backend.configuration;
+
+import com.znane_groby.backend.model.City;
+import com.znane_groby.backend.model.Graveyard;
+import com.znane_groby.backend.repository.CityRepository;
+import com.znane_groby.backend.repository.GraveyardRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+@Configuration
+@Slf4j
+public class LoadDatabase {
+    GraveyardRepository graveyardRepository;
```

Comment:
Ciekawostka: musimy się zastanowić graveyard czy cemetery (uwaga, cemetery bez 'n' w środku) https://www.quickanddirtytips.com/education/grammar/cemetery-versus-graveyard

Reply:
W sumie pierwszy raz widzę to cemetery, ale jak pisze ta laska jest nawet bardziej popularne, więc może być (jest jedną literę krótszy)